DfE.FindInformationAcademiesTrusts/Pages/accessibility.cshtml

```diff
@@ -0,0 +1,125 @@
+    </ul>
+    <p class="govuk-body">
+      We’ve also made the website text as simple as possible to understand.
+    </p>
+    <p class="govuk-body">
+      <a href="https://mcmw.abilitynet.org.uk" class="govuk-link" rel="noopener" target="_blank">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.
+    </p>
+    <h2 class="govuk-heading-m">
+      How accessible this website is
+    </h2>
+    <p class="govuk-body">
+      We know some parts of this website are not fully accessible:
+    </p>
+    <p class="govuk-body">
+      This section will be updated after the accessibility audit has taken place.
+    </p>
+    <h2 class="govuk-heading-m">
+      Give feedback and contact us
+    </h2>
+    <p class="govuk-body">
+      If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille email <a href="mailto:regionalservices.rg@education.gov.uk." class="govuk-link" rel="noopener" target="_blank">regionalservices.rg@education.gov.uk.</a>
+    </p>
+    <p class="govuk-body">
+      We’ll consider your request and get back to you in 5 working days.
+    </p>
+    <h2 class="govuk-heading-m">
+      Reporting accessibility problems with this website
+    </h2>
+    <p class="govuk-body">
+      We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact:
+      <a href="mailto:regionalservices.rg@education.gov.uk." class="govuk-link" rel="noopener" target="_blank">regionalservices.rg@education.gov.uk.</a>
+    </p>
+    <h2 class="govuk-heading-m">
+      Enforcement procedure
+    </h2>
+    <p class="govuk-body">
+      The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact:
+      <a href="https://www.equalityadvisoryservice.com/" class="govuk-link" rel="noopener" target="_blank">The Equality Advisory and Support Service (EASS) (opens in new tab)</a>.
+    </p>
+    <h2 class="govuk-heading-m">
+      Technical information about this website’s accessibility
+    </h2>
+    <p class="govuk-body">
+      Department for Education is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+    </p>
+    <h3 class="govuk-heading-s">
+      Compliance status
+    </h3>
+    <p class="govuk-body">
+      This section will be updated after the accessibility audit has taken place.
+    </p>
+    <h2 class="govuk-heading-m">
+      Non-accessible content
+    </h2>
+    <p class="govuk-body">
+      The content listed below is non-accessible for the following reasons.
```

Comment:
I think this needs to be 'This section will be updated after the accessibility audit has taken place' ?

Reply:
yes..the headers are to be updated also..good spot!